learning_journal_basic1/routes.py

```diff
@@ -1,3 +1,10 @@
+"""This is the routes file."""
+
+
 def includeme(config):
-    config.add_static_view('static', 'static', chache_max_age=3600)
-    config.add_route('home', ' / ')
+    """Add routes to Pyramid's Configurator."""
+    config.add_static_view('static', 'static', cache_max_age=3600)
+    config.add_route('list_view', '/')
+    config.add_route('detail_view', '/journal/{id:\d+}')
+    config.add_route('entry_view', '/journal/new_entry')
+    config.add_route('edit_view', '/journal/edit_view/{id:\d+}')
```

Comment:
These route names should be more semantic to the types of pages that you want linked to, not the views that render them. Multiple views can hit a single route. Multiple routes can hit a single view. Make the route names semantic to the actual pages themselves.